### ISSUE TITLE
Create Hawt.io plugin to browse data sent to the Message Store Cloudlet

### ISSUE THREAD

**Issue by hekonsek** [rhiot/rhiot#34]

**Comment by ajurasz** [comment]
@hekonsek still in progress.

So far I have made few modification to document service to get list of all db collections through <code>/mongodb</code> endpoint. Another think was a new operation <code>findAll</code>

I have got plugin UI for selecting collections and displaying all it's data in the table.

But I don't like result of the table this is because collection structure is dynamic so it can change in the middle, so what I will do is to create html table structure base on first document in collection and then if it changes then sorry but some of new fields won't be visible in table.

When I will resolve the issue with displaying mongo collections in html tables then I will need to:
-create docker image
-migrate to Typescript (for now it is faster for me to use JavaScript
-deploy plugin
-merge branch to master
